lib/tasks/one_off.rb

```diff
@@ -20,4 +20,17 @@
     puts('Add the following public key to Github account with access to this repo:')
     puts("\n#{project.ssh_public_key}\n")
   end
+
+  desc 'Backfills projects to use diggit-bot github token'
```

Comment:
May not need this backfill, depending on whether it seems sensible to stick with diggit-bot as a default client.